src/openforms/emails/templatetags/appointments.py

```diff
@@ -22,7 +22,7 @@ def appointment_information(context):
     else:
         template_name = "emails/templatetags/appointment_information.html"
 
-    # check for new style appointments
+    # check for appointments
     submission = context["_submission"]
     appointment = get_appointment(submission)
     plugin_id = appointment.plugin if appointment else ""
```

Comment:
Shouldn't this now basically be an `assert appointment` because it doesn't make sense to use this tag if there's no appointment?